compiler/semantic/sql.go

```diff
@@ -322,35 +325,141 @@ func (a *analyzer) semSelectValue(sel *ast.Select, sch schema, seq dag.Seq) (dag
 	return seq, &dynamicSchema{}
 }
 
+func (a *analyzer) semValues(values *ast.Values, seq dag.Seq) (dag.Seq, schema) {
+	var schema *super.TypeRecord
+	exprs := make([]dag.Expr, 0, len(values.Exprs))
+	sctx := super.NewContext()
+	for _, astExpr := range values.Exprs {
+		e := a.semExpr(astExpr)
+		val, err := kernel.EvalAtCompileTime(sctx, e)
+		if err != nil {
+			a.error(astExpr, errors.New("expressions in values clause must be constant"))
+			return seq, badSchema()
+		}
+		recType, ok := super.TypeUnder(val.Type()).(*super.TypeRecord)
+		if !ok {
+			// Turn non record values into single column tuple
+			f := &dag.Field{Kind: "Field", Name: "c0", Value: e}
+			e = &dag.RecordExpr{Kind: "RecordExpr", Elems: []dag.RecordElem{f}}
+			val, _ := kernel.EvalAtCompileTime(sctx, e)
```

Comment:
I know this should never fail but that could always change down the road and we want to know if it does.
```suggestion
			val, err := kernel.EvalAtCompileTime(sctx, e)
			if err != nil {
				panic(err)
			}
```